site/content/cluster/binaries_cluster.md

```diff
@@ -0,0 +1,264 @@
+{{% /notice %}}
+
+This example creates three static IP addresses for three storage nodes, using the embedded coordinator.
+
+This guide assumes you have host names configured, i.e., running `hostname` on a host in the cluster returns the host ID you use when creating the cluster placement.
+
+{{% notice tip %}}
+When using GCP the name of your instance is the host name. When you create an instance, click _Management, disks, networking, SSH keys_, under _Networking_, click the default interface, click the _Primary internal IP_ drop down, select _Reserve a static internal IP address_, give it an appropriate name and description, and use _Assign automatically_.
+{{% /notice %}}
+
+{{% notice tip %}}
+When using AWS, you can use the host name supplied for the provisioned VM as your host ID, or use the `environment` host ID resolver and pass the host ID when launching the database process with an environment variable.
+{{% /notice %}}
+
+For example, if you used `M3DC_HOST_ID` for the environment variable name, use the following in your configuration:
+
+```yaml
+hostID:
+  resolver: hostname
+    - ${M3DC_HOST_ID:""}
+```
+
+Then start the `m3dbnode` process with:
+
+{{< tabs name="start_container" >}}
+{{% tab name="Pre-built binary" %}}
+
+```shell
+M3DB_HOST_ID=m3db001 m3dbnode -f <config-file.yml>
+```
+
+{{% notice info %}}
+Depending on your operating system setup, you might need to prefix the command with `sudo`.
+{{% /notice %}}
+
+{{% /tab %}}
+{{% tab name="Output" %}}
+
+<!-- TODO: Perfect image, pref with terminalizer -->
+
+<!-- TODO: Update image -->
+
+![Docker pull and run](/docker-install.gif)
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Kernel Configuration
+
+Depending on the default limits of your bare-metal machine or VM, M3 may need some Kernel tweaks to run as efficiently as possible, and [we recommend you review those](/docs/operational_guide/kernel_configuration) before running M3 in production.
+
+## Configuration files
+
+You configure each M3 component by passing the location of a YAML file with the `-f` argument.
+
+The file groups configuration into `coordinator` or `db` sections that represent the `M3Coordinator` and `M3DB` instances of single-node cluster.
+
+{{% notice tip %}}
+You can find more information on configuring M3DB in the [operational guides section](/docs/operational_guide/).
+{{% /notice %}}
+
+{{% notice note %}}
+The steps in this guide have the following 3 seed nodes, you need to change your configuration to suit the details of yours, including the details of an etcd cluster in the `etcdClusters` section of the M3 configuration file.
```

Comment:
`etcdClusters` has now become `endpoints` when using discovery type `m3dbCluster`. 

Comments I left below describe it better (hopefully! if not please do ask questions of course).